pkgs/data/fonts/material-design-icons/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "material-design-icons-${version}";
+  version = "3.2.89";
+
+  src = fetchFromGitHub {
+    owner  = "Templarian";
+    repo   = "MaterialDesign-Webfont";
+    rev    = "v${version}";
+    sha256 = "1rxaiiij96kqncsrlkyp109m36v28cgxild7z04k4jh79fvmhjvn";
+  };
+
+  buildCommand = ''
```

Comment:
This should be `buildPhase` to make use of the default builder.

Reply:
Repo has no Makefile, so I think it should be `installPhase` instead as the default implementation executes `make install`